README.md

```diff
@@ -65,7 +65,7 @@ We're back! Use this repo to share and keep track of software, tech, CS, PM, qua
 | **[Atlassian](https://simplify.jobs/c/Atlassian)** | Finance & Accounting Intern - 2024 Summer U.S | Mountain View, CA | <a href="https://jobs.lever.co/atlassian/cfb254a3-e3d9-4666-b425-92b2744db34d/apply?utm_source=Simplify&ref=Simplify"><img src="https://i.imgur.com/w6lyvuC.png" width="84" alt="Apply"></a> <a href="https://simplify.jobs/p/6e36fb02-fde8-4cbf-95de-10e273cc5ea5?utm_source=GHList"><img src="https://i.imgur.com/aVnQdox.png" width="30" alt="Simplify"></a> | Sep 08 |
 | **[Northwestern Mutual](https://simplify.jobs/c/Northwestern-Mutual)** | Investment Data – Analytics & Research Internship | Milwaukee, WI | <a href="https://northwesternmutual.wd5.myworkdayjobs.com/corporate-careers/job/Milwaukee-WI-Corporate/Investment-Data--Analytics---Research-Internship_JR-39422-1?utm_source=Simplify&ref=Simplify"><img src="https://i.imgur.com/w6lyvuC.png" width="84" alt="Apply"></a> <a href="https://simplify.jobs/p/e01e7595-3268-4b6d-9215-93a0959fae2f?utm_source=GHList"><img src="https://i.imgur.com/aVnQdox.png" width="30" alt="Simplify"></a> | Sep 08 |
 | **[Spring Health](https://simplify.jobs/c/Spring-Health)** | Software Engineer Intern | Remote | <a href="https://boards.greenhouse.io/springhealth66/jobs/4315743005?utm_source=Simplify&ref=Simplify"><img src="https://i.imgur.com/w6lyvuC.png" width="84" alt="Apply"></a> <a href="https://simplify.jobs/p/01ce30c8-b5f3-4319-b280-6a8b561d0055?utm_source=GHList"><img src="https://i.imgur.com/aVnQdox.png" width="30" alt="Simplify"></a> | Sep 08 |
-| **[Wolverine Trading](https://simplify.jobs/c/wolverine-trading)** | Legal and Compliance Intern - Summer 2024 | Chicago, IL | <a href="https://jobs.lever.co/wolve/50397c44-c2df-4370-8d29-db6d48ca6e09/apply?utm_source=Simplify&ref=Simplify"><img src="https://i.imgur.com/w6lyvuC.png" width="84" alt="Apply"></a> <a href="https://simplify.jobs/p/8a1e51a5-75a8-485c-b023-55f5b7727d7a?utm_source=GHList"><img src="https://i.imgur.com/aVnQdox.png" width="30" alt="Simplify"></a> | Sep 08 |
+| **[Wolverine Trading](https://simplify.jobs/c/wolverine-trading)** | Software Engineer Intern - Summer 2024 | Chicago, IL | <a href="https://jobs.lever.co/wolve/30de7311-5504-4092-afbb-061e61bb7318/apply?utm_source=Simplify&ref=Simplify"><img src="https://i.imgur.com/w6lyvuC.png" width="84" alt="Apply"></a> <a href="https://simplify.jobs/p/8a1e51a5-75a8-485c-b023-55f5b7727d7a?utm_source=GHList"><img src="https://i.imgur.com/aVnQdox.png" width="30" alt="Simplify"></a> | Sep 08 |
```

Comment:
Needs to be updated via their issue template. Refer: https://github.com/SimplifyJobs/Summer2024-Internships/blob/dev/CONTRIBUTING.md#editing-an-internship